src/org/traccar/Context.java

```diff
@@ -256,6 +257,13 @@ public static void init(String[] arguments) throws Exception {
                     Context.getConfig().getString("mail.templatesPath", "templates/mail") + "/");
             velocityProperties.setProperty("runtime.log.logsystem.class",
                     "org.apache.velocity.runtime.log.NullLogChute");
+
+            String address = config.getString("web.address", InetAddress.getLocalHost().getHostAddress());
+            String port = config.getString("web.port", "8082");
+            String webUrl = "http://" + address + ":" + port + "/";
```

Comment:
Probably we can use some URL builder to construct this. It will handle automatically cases like when port is 80, we don't need to add it.